packages/pwa/app/utils/url.js

```diff
@@ -103,62 +119,68 @@ export const searchUrlBuilder = (searchTerm) => `/search?q=${searchTerm}`
 
 /**
  * Returns a relative URL for a locale short code.
+ * This function will take based on the url config from the pwa-kit.config
+ * to look for the locale position from the path/param
+ * and replace it with new shortCode
```

Comment:
Looks like there is a grammar mistake here. Also we'll want to not reference pwa-kit.config file. Maybe we can simply refer to it as "your application configuration" so we don't have to make changes like this in the future if the file name/location change.